user/service/client/client.go

```diff
@@ -202,6 +204,23 @@ func (c *Client) Delete(ctx context.Context, id string, deleet *user.Delete, con
 			logger.WithError(err).Error("Unable to destroy profile")
 		}
 	}
+	//Sending delete user event to kafka
+	userName := *result.Username
+	logger.Infof("preparing to send delete user kafka message for %v", userName)
+	if !strings.HasSuffix(userName, "@tidepool.io") && !strings.HasSuffix(userName, "@tidepool.org") {
+		var role []string
+		if result != nil && result.Roles != nil {
+			role = *result.Roles
+		}
+		eventType := "delete-user"
```

Comment:
I suggest you define this as `const DELETE_USER_EVENT_TYPE="delete-user"` at the top of this file

Reply:
will do

Reply:
Actually, the linter suggests `DeleteUserEventType`